SU-SWS/stanford_profile
# D8CORE-2295: adding spacing for buttons in a row in the wysiwyg

# NOT READY FOR REVIEW

# Summary
- Adding MS 0 to multiple buttons in the WYSIWYG.

# Needed By (Date)
- 7/7

# Urgency
- Medium

# Steps to Test

1. Pull in this branch.
2. add buttons in the text editor on a page.
3. Notice there is padding between multiple buttons in a row. 

# Affected Projects or Products
- Core

# Associated Issues and/or People
- D8CORE-2295
- @kerri-augenstein
